### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by GlydetheSurfer** [FreeTubeApp/FreeTube#4108]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Go to Settings
2. Click on Tor/Proxy
3. Click on Enable Tor / Proxy and Test Proxy with Tor expert bundle 100% loaded (administrator privileges).
4. See "Error getting network information. Is your proxy configured properly?"

### Expected Behavior

I expect to see a different location than where I'm located show up. (or a different ip address)

### Issue Labels

usability issue

### FreeTube Version

v0.19.0 Beta

### Operating System Version

Windows 11 Version 22H2

### Installation Method

.exe

### Primary API used

Local API

### Additional Information

![image](https://github.com/FreeTubeApp/FreeTube/assets/32045132/4577b379-17c6-444c-8804-d75e7d483086)
![image](https://github.com/FreeTubeApp/FreeTube/assets/32045132/3c7f5771-1633-4fe2-8266-9a5fff4f0df4)

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by patheticcockroach** [comment]
In case this helps, I have the same issue and, in my case:

1. the connection actually works fine, it's just the "test proxy" feature that doesn't
2. I observed the same problem with other SOCKS proxies that I regularly use (also running on localhost)

(using "v0.19.1 Beta" on Windows 10)

**Comment by GlydetheSurfer** [comment]
> In case this helps, I have the same issue and, in my case:
> 
> 1. the connection actually works fine, it's just the "test proxy" feature that doesn't
> 2. I observed the same problem with other SOCKS5 proxies that I regularly use (also running on localhost)
> 
> (using "v0.19.1 Beta" on Windows 10)

It's good that this gets fixed anyway. I want to be certain that I'm actually connected to the proxy and not just exposed on my public ip address

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@patheticcockroach @GlydetheSurfer so if i understand correctly its just the test that is going wrong but u are still able to play videos through tor?

**Comment by patheticcockroach** [comment]
> @patheticcockroach @GlydetheSurfer so if i understand correctly its just the test that is going wrong but u could confirm that u are still able to play videos through tor in FT?

Indeed. And I verified that by shutting down Tor (or my other SOCKS5 proxy) and observing that in such a case nothing loads.

I guess that, for the most paranoid, it doesn't rule out the possibility that the connection still ignores the proxy but only works if the proxy is up, but that would be a bit weird 🤔

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Thanks for your reply, yeah i think this would make some people really really confused.

If u have some spare time could u maybe look into https://github.com/FreeTubeApp/FreeTube/discussions/4125? I myself dont have allot of experience with setting up tor with FT

**Comment by GlydetheSurfer** [comment]
> > @patheticcockroach @GlydetheSurfer so if i understand correctly its just the test that is going wrong but u could confirm that u are still able to play videos through tor in FT?
> 
> Indeed. And I verified that by shutting down Tor (or my other SOCKS5 proxy) and observing that in such a case nothing loads.
> 
> I guess that, for the most paranoid, it doesn't rule out the possibility that the connection still ignores the proxy but only works if the proxy is up, but that would be a bit weird 🤔

i can confirm it does connect to the proxy even though it shows the error.. as when i opened a video it took awhile to load, and when it did, it appeared to be pixelated at first, indicating that the connection was slow. it doesnt do that when im not connected to a proxy as i have fast internet.

**Comment by absidue** [comment]
Looks like the API that we are using for the proxy test has changed and it now redirects you to the API docs, guess we need to update our code to support the new setup or switch to a different API.
